protobufs/entity/service.proto

```diff
@@ -131,6 +131,8 @@ message RelationshipData {
   string owner_identity = 1;
```

Comment:
Do we need the `owner_identity`?

We already have the owner DID (selfDID) from the context or is there a case where `owner_identity != selfDID`?